TCAT/Cells/RouteTableViewCell.swift

```diff
@@ -144,33 +143,33 @@ class RouteTableViewCell: UITableViewCell {
     }
     
     func setLiveElements(withFirstDepartDirection firstDepartDirection: Direction?) {
+        
         if let direction = firstDepartDirection {
-            Network.getBusLocations(routeID: String(direction.routeNumber), tripID: direction.tripID, stopID: direction.startLocation.id)
-                .perform(withSuccess: { (result) in
-                    if let busLocation = result.busLocation {
+            
+            Network.getBusLocations([direction]).perform(withSuccess: { (result) in
+                
+                for busLocation in result.busLocations {
```

Comment:
why is this a for loop? i only want the live update for the first depart direction so i would think i would only need `result.busLocations.first`. im just confused on why we're going through an array of `busLocation` when I really only want the delay for the bus that corresponds to the first depart direction.

Reply:
Resolved with new `/delay` endpoint